### PR TITLE
Alias preview urls

### PR DESCRIPTION
**Pull request by kyle-ssg**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/deployment/locally-api#pre-commit) to check linting
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?

## Changes

Branches now get a url alias for staging / production preview deployments. Prior to this the url would change per commit.

## How did you test this code?

See new preview urls

### CODE REVIEW

**Comment by dabeeeenster** [comment]
What's this good for?!

**Comment by kyle-ssg** [comment]
It means that URLs are consistent / don't change per commit.

**Comment by dabeeeenster** [comment]
> It means that URLs are consistent / don't change per commit.

ah great

File: frontend/package.json [hunk 1]
```diff
@@ -119,6 +119,7 @@
     "suppress-exit-code": "^1.0.0",
     "terser-webpack-plugin": "^5.3.6",
     "toml": "^3.0.0",
+    "vercel": "^28.16.15",
```
Comment (dabeeeenster) [inline frontend/package.json:122]: Should this not be in devDependencies?

Comment (kyle-ssg) [reply frontend/package.json:122]: No, because our actions do npm ci

Comment (kyle-ssg) [reply frontend/package.json:122]: It's the same reason why webpack etc is there too.

**Review by dabeeeenster — APPROVED** [review]

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**